datalad/distribution/install.py

```diff
@@ -725,6 +731,10 @@ def __call__(
                     else:
                         installed_items.append(rec_installed)
 
+                except Exception:
+                    # Error itself should already be logged.
+                    lgr.info("{0} skipped.".format(subds))
```

Comment:
Nope. For the same reason as before. This is dealt with in the recursive instance. It's just about skipping.